tests.py

```diff
@@ -182,19 +182,19 @@ def test_noise_sfts(self):
             SFTWindowType=self.SFTWindowType,
             SFTWindowBeta=self.SFTWindowBeta,
             sqrtSX=self.sqrtSX,
-            **default_signal_params,
+            Band=0.5,
+            **self.signal_parameters,
         )
         noise_and_signal_writer.make_data(verbose=True)
         # FIXME: here and everywhere below,
         # get_sft_array() only looks at first detector
         times, freqs, data = pyfstat.helper_functions.get_sft_array(
             noise_and_signal_writer.sftfilepath
         )
+        # Store the maximum SFT values for later comparison
         max_values_noise_and_signal = np.max(data, axis=0)
         max_freqs_noise_and_signal = freqs[np.argmax(data, axis=0)]
-        self.assertTrue(len(times) == int(np.ceil(self.duration / self.Tsft)))
-        # with signal: all SFTs should peak at same freq
-        self.assertTrue(len(np.unique(max_freqs_noise_and_signal)) == 1)
+        self.assertTrue(len(times) == self.duration / self.Tsft)
```

Comment:
Can we keep the same-bin test commented out, with a FIXME that it should account for bin drift to be properly useful?
Also again the `np.ceil` should stay I think.